### PR TITLE
Spelling bee Assignment

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,37 @@
 ﻿namespace Training {
     internal class Program {
         static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
+         string[] words = File.ReadAllLines ("C:\\Users\\priyanka.s\\Downloads\\words.txt");
+         char[] letters = { 'U', 'X', 'L', 'T', 'A', 'E', 'N' };
+         int total = 0, total2 = 0, total1 = 0, index = 0; ;
```
Comment (MetaNaveen) [inline Program.cs:6]: Extra semicolon at the end of line found. Remove it.

Comment (MetaNaveen) [inline Program.cs:6]: Variable names could be more descriptive/meaningful.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,37 @@
 ﻿namespace Training {
     internal class Program {
         static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
+         string[] words = File.ReadAllLines ("C:\\Users\\priyanka.s\\Downloads\\words.txt");
+         char[] letters = { 'U', 'X', 'L', 'T', 'A', 'E', 'N' };
+         int total = 0, total2 = 0, total1 = 0, index = 0; ;
+         string[] wordarray = new string[words.Length];
+         int[] point = new int[words.Length];
+         foreach (string word in words) {
```
Comment (MetaNaveen) [inline Program.cs:9]: If you logic requires index, you could use for loop instead of foreach.

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
+                  } else
+                     marks = word.Length;
+                  total2 = total2 + marks;
+               }
+               point[index] = marks;
+               wordarray[index] = word;
+               index++;
+            }
+         }
+         Array.Sort (point, wordarray, 0, index, Comparer<int>.Create ((a, b) => b.CompareTo (a)));
```
Comment (MetaNaveen) [inline Program.cs:27]: It seems the output is not complete. Attaching my solution for reference. Kindly make sure of the correctness.

File: Program.cs (hunk 1 above)
Comment (Priyanka15-12) [reply Program.cs]: Removed

File: Program.cs (hunk 2 above)
Comment (Priyanka15-12) [reply Program.cs:29]: Changes are done

File: Program.cs [hunk 4]
```diff
@@ -1,7 +1,55 @@
...
+                     marks += word.Length + 7;
+                  } else marks += word.Length;
+               }
+               (int, string) list = (marks, word);
+               wordsList.Add (list);
+            }
+         }
+         var descendingOrderedList = wordsList.OrderByDescending (x => x.Item1).ToList ();
+         for (int i = 0; i < descendingOrderedList.Count; i++) {
+            Console.WriteLine ($"{descendingOrderedList[i].Item1,3}. {descendingOrderedList[i].Item2}");
```
Comment (MetaNaveen) [inline Program.cs:46]: What if the result contains two or more words with the same score? Can you color those as well in GREEN color? 
(To test: add "TNATLUXE" (reversse of EXULTANT) and try)

**Review by MetaNaveen — CHANGES_REQUESTED** [review]
Make the changes. (Also share the words.txt file by adding it in the project)

File: Program.cs [hunk 5]
```diff
@@ -1,7 +1,55 @@
...
+            Console.ResetColor ();
```
[the remark this answers is not in the document]
Comment (Priyanka15-12) [reply Program.cs]: changes are done

File: Program.cs [hunk 6]
```diff
@@ -1,7 +1,55 @@
...
+         char[] letters = { 'U', 'X', 'L', 'T', 'A', 'E', 'N' };
+         List<(int, string)> wordsList = new ();
+         foreach (string word in words) {
+            if (word.Length >= 4 && word.Contains (letters[0]) && word.All (letters.Contains)) {
+               int marks = 0;
+               if (word.Length == 4)
+                  marks += 1;
+               else if (word.Length > 4) {
+                  if (letters.All (word.Contains)) {
+                     Console.ForegroundColor = ConsoleColor.Green;
```
[the remark this answers is not in the document]
Comment (Priyanka15-12) [reply Program.cs]: Changes are done.

**Pull request closed without merging**